src/Mutator/Util/AbstractIdenticalComparison.php

```diff
@@ -109,12 +121,58 @@ private function getReturnType(Expr\FuncCall $call): ?string
         }
 
         $reflection = new ReflectionFunction($name);
-        $returnType = $reflection->getReturnType();
 
-        if (!$returnType instanceof ReflectionNamedType) {
-            return $this->reflectionCache[$name] = null;
+        return $this->reflectionCache[$name] = $reflection->getReturnType();
+    }
+
+    /**
+     * @param Node\Scalar|Expr\ConstFetch|Expr\FuncCall $expr
+     */
+    private function narrowReturnType(ReflectionType $returnType, Expr $expr): ?string
```

Comment:
just for more clarity, what do you think about renaming `$returnType` to something like `$functionCallReturnType`?

Just to minimize the information needed to be kept in the head reading the code and to ease the understanding